projects/packages/search/package.json

```diff
@@ -0,0 +1,86 @@
+{
+	"name": "jetpack-search",
+	"version": "0.2.0-alpha",
+	"description": "Package for Jetpack Search products",
+	"main": "main.js",
+	"directories": {
+		"test": "tests"
+	},
+	"scripts": {
+		"build": "pnpm run build-development",
+		"build-production": "NODE_ENV=production BABEL_ENV=production pnpm run clean-all && pnpm run build-production-instant && pnpm run build-production-customberg && pnpm run build-production-dashboard && pnpm run build-production-php",
+		"build-production-instant": "echo 'Not yet implemented.'",
+		"build-production-customberg": "echo 'Not yet implemented.'",
+		"build-production-dashboard": "echo 'Not yet implemented.'",
+		"build-production-php": "COMPOSER_MIRROR_PATH_REPOS=1 COMPOSER_ROOT_VERSION=dev-master composer install -o --no-dev --classmap-authoritative --prefer-dist",
+		"build-development": "NODE_ENV=development BABEL_ENV=development pnpm run clean-all && pnpm run build-instant && pnpm run build-customberg && pnpm run build-dashboard && pnpm run build-php",
+		"build-instant": "echo 'Not yet implemented.'",
+		"build-customberg": "echo 'Not yet implemented.'",
+		"build-dashboard": "echo 'Not yet implemented.'",
+		"build-php": "composer install --ignore-platform-reqs",
```

Comment:
This won't work as you intend, the environment variables when set this way only apply to the command they're attached to, not later commands after `&&`. What you'd have to do is like
```
NODE_ENV=production BABEL_ENV=production pnpm run clean-all && NODE_ENV=production BABEL_ENV=production pnpm run build-production-instant && NODE_ENV=production BABEL_ENV=production pnpm run build-production-customberg && NODE_ENV=production BABEL_ENV=production pnpm run build-production-dashboard && NODE_ENV=production BABEL_ENV=production pnpm run build-production-php"
```
Which is obviously kind of ridiculous. What I'd suggest instead is something like
```suggestion
		"build-development": "NODE_ENV=development BABEL_ENV=development pnpm run build",
		"build-production": "NODE_ENV=production BABEL_ENV=production pnpm run build && pnpm exec validate-es -- build/",
		"build": "pnpm run clean-all && pnpm run build-instant && pnpm run build-customberg && pnpm run build-dashboard && pnpm run build-php",
		"build-instant": "echo 'Not yet implemented.'",
		"build-customberg": "echo 'Not yet implemented.'",
		"build-dashboard": "echo 'Not yet implemented.'",
		"build-php": "composer install --ignore-platform-reqs",
```
That way the environment variables apply to the whole `pnpm run build` and everything inside without having to repeat them. I also added a call to the `validate-es` tool that helps check that the build properly transpiled everything that might need transpilation to match the browserslist config; you might need to set a config to avoid browserslist's default that still includes IE11.